ocs_ci/deployment/deployment.py

```
@@ -4060,6 +4060,388 @@ def _patch_ramen_csv(self, csv_prefix, namespace, new_image):
+
+        """
+        old_ctx = config.cur_index
+        clusters_config = {}
+
+        try:
+            config.switch_acm_ctx()
+            hub_kubeconfig = config.RUN.get("kubeconfig")
+
+            if hub_kubeconfig and os.path.exists(hub_kubeconfig):
+                hub_dest = os.path.join(ramenctl_dir, "hub.yaml")
+                shutil.copy(hub_kubeconfig, hub_dest)
```

Ruff (0.16.1)

[error] Undefined name `shutil`

(F821)